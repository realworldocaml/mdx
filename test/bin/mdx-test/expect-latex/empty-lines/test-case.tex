This shell block contains an empty line within a padded block:

\begin{sh}
 $ echo ''

\end{sh}

\begin{sh}
  $ echo "Hello..." && echo "" && echo "world!"
\end{sh}

This toplevel block contains an empty line within a padded block:

\begin{ocaml}
 # print_newline;;

\end{ocaml}
