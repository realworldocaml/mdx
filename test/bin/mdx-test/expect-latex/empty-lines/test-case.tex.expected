This shell block contains an empty line within a padded block:

\begin{sh}
 $ echo ''
 
\end{sh}

\begin{sh}
  $ echo "Hello..." && echo "" && echo "world!"
  Hello...
  
  world!
\end{sh}

This toplevel block contains an empty line within a padded block:

\begin{ocaml}
 # print_newline;;
 - : unit -> unit = <fun>
\end{ocaml}
